tests/.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
Any reason we need this file that we can do in the original top level .`eslintrc.js`?